### ISSUE TITLE
argparse: show choices once per argument

### ISSUE THREAD

**Issue by b71ed331-7af2-49a6-8352-0f9a31cd6e18** [python/cpython#86424]
BPO | [42258](https://bugs.python.org/issue42258)
--- | :---
Nosy | @rhettinger, @shihai1991, @mendelmaleh
PRs | <li>python/cpython#23143</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2020-11-04.11:21:13.911>
labels = ['type-bug', 'library', '3.10']
title = 'argparse: show choices once per argument'
updated_at = <Date 2020-11-11.16:50:00.031>
user = 'https://github.com/mendelmaleh'
```

bugs.python.org fields:
```python
activity = <Date 2020-11-11.16:50:00.031>
actor = 'mendelmaleh'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2020-11-04.11:21:13.911>
creator = 'mendelmaleh'
dependencies = []
files = []
hgrepos = []
issue_num = 42258
keywords = ['patch']
message_count = 2.0
messages = ['380509', '380771']
nosy_count = 5.0
nosy_names = ['rhettinger', 'python-dev', 'paul.j3', 'shihai1991', 'mendelmaleh']
pr_nums = ['23143']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue42258'
versions = ['Python 3.10']
```

</p></details>

**Comment by savannahostrowski** [comment]
The original 3.10 behaviour no longer reproduces on 3.13 and main, so the suggested behaviour is now implemented (I believe it was this commit c4a2e8a2c5188c3288d57b80852e92c83f46f6f3). I wouldn't consider this a bug so I don't think it'd be eligible for backport to 3.12 and earlier.

@serhiy-storchaka I wanted to tag you here since you merged the PR associated with the commit mentioned above that updated this behaviour. If you agree, I believe this can be closed!

**Comment by serhiy-storchaka** [comment]
Agree with @savannahostrowski. Thank you for noticing this.
